jobs.py

```diff
@@ -146,8 +146,18 @@ def update_invoice(stub, invoice, db_invoice):
     db_invoice.save()
 
 def update_forwards(stub):
-    records = Forwards.objects.count()
-    forwards = stub.ForwardingHistory(ln.ForwardingHistoryRequest(start_time=1420070400, index_offset=records, num_max_events=100)).forwarding_events
+    latest_forward = Forwards.objects.order_by('-forward_date', '-id').first()
+    start_time = int(latest_forward.forward_date.timestamp()) if latest_forward else 1420070400
+    processed_count = Forwards.objects.filter(forward_date=latest_forward.forward_date).count() if latest_forward else 0
```

Comment:
could we use latest_forward.id here instead of another DB call for count()?

Reply:
That was the initial approach. However, I seem to remember this was not reliable when we tried– For example, if the `gui_forwards` table had ever been cleared in the past (using either the `/reset` page or manually, without also reseting the id counter), the id would not match the actual # of records in the table, leading to a mismatch between the id and the actual count of rows in the table.

Using `Forwards.objects.count()` will always give the exact number of records in the table, regardless of whether the id sequence was reset or if any gaps exist. While slightly less efficient, it should be more accurate and reliable. Thoughts?